apiary.apib

```diff
@@ -175,22 +221,83 @@ Note: Rewards generated in the Sandbox environment will be delivered to the API
 
     + Body
       {
-          "order": {
-            "id": "QABSTARTSFSIO",
-            "external_id": "[ID FROM CLIENT]",
-            "payment": {
-              "subtotal": order.amount_dollars,
-              "total": order.price_dollars,
-              "fees": order.fee_dollars
-            },
-            "rewards": [{
-              id: "DDABSUKSFSTY"
-            }]
-          }
+        "order": {
+          "id": "QABSTARTSFSIO",
+          "external_id": "[ID FROM CLIENT]",
+          "status": "EXECUTED",
+          "payment": {
+            "subtotal": order.amount_dollars,
+            "total": order.price_dollars,
+            "fees": order.fee_dollars
+          },
+          "rewards": [{
+            id: "DDABSUKSFSTY"
+          }]
+        }
+      }
+
++ Response 201 (application/json)
+    + Attributes (object)
+        + order (object)
+          + id (string)
+          + external_id (string)
+          + status (string)
+          + rewards (array)
+            + (object)
+              + id (string)
+          + payment (object)
+              + subtotal (number)
+              + fees (number)
+              + total (number)
+
+    + Body
+      {
+        "order": {
+          "id": "QABSTARTSFSIO",
+          "external_id": "[ID FROM CLIENT]",
+          "status": "EXECUTED",
+          "payment": {
+            "subtotal": order.amount_dollars,
+            "total": order.price_dollars,
+            "fees": order.fee_dollars
+          },
+          "rewards": [{
+            id: "DDABSUKSFSTY"
+          }]
+        }
+      }
+
++ Response 402 (application/json)
+    + Attributes (object)
+        + errors (object)
+          + message (string)
+
+    + Body
+      {
+        "errors": {
+          "message": "Order failed: payment error"
+        }
+      }
+
++ Response 400 (application/json)
+    + Attributes (object)
+        + errors (object)
+          + message (string)
+
+    + Body
+      {
+        "errors": {
+          "message": 'Order failed: validation failure'
+        }
```

Comment:
think we also have a `payload` key where the validation failure is sent, not sure if important to have here